src/js/Hls.js

```diff
@@ -504,7 +512,9 @@ class Hls extends Meister.MediaPlugin {
 
     // HLS errors
     onError(e, data) {
-        console.warn(`Error in ${this.name}, type: ${data.details}, will attempt to recover.`);
+        this.amountOfErrors += 1;
```

Comment:
We're just counting any kind of error? That doesn't seem right to me. I guess we're trying to measure the bufferStalled error, so it's probably best to specifically check for that. That way we can also make the error message more informative.